isis/tests/TgoCassisModuleTests.cpp

```diff
@@ -138,7 +138,7 @@ TEST_F(TgoCassisModuleKernels, TgoCassisStitchUnstitch) {
   cubeList->write(cubeListFile);
 
   QVector<QString> stitchArgs = {"fromlist=" + cubeListFile,
-                "outputprefix=" + prefix.path() + "/stitched"};
+                "outputprefix=" + prefix.path() + "/stitched-"};
```

Comment:
The additional dashes are still an issue here. If we have to add to the output path here this is considered a breaking change. Rather than change the tests, we need to make the code conform to the tests

Reply:
addressed with latest commits